crates/symbolicator-js/src/symbolication.rs

```diff
@@ -307,7 +316,11 @@ async fn symbolicate_js_frame(
 }
 
 fn apply_source_context(frame: &mut JsFrame, source: &str) -> Result<(), JsModuleErrorKind> {
-    let lineno = frame.lineno as usize;
+    if frame.lineno.is_none() {
+        return Ok(());
+    }
```

Comment:
```suggestion
    let Some(lineno) = frame.lineno else {
        return Ok(());
    }
```

Don't have to `unwrap_or_default` afterwards then.